mscore/plugin/qmledit.cpp

```diff
@@ -375,27 +375,6 @@ QmlEdit::QmlEdit(QWidget* parent)
       highlightCurrentLine();
       }
 
-//---------------------------------------------------------
```

Comment:
Moved to pluginCreator.cpp

Reply:
The focus sets the Musescore state, toggling between disabled and not disabled, so it's the whole plugin Creator that should handle these state switches, not the qml edit.